js/mva/js/NetworkDesigner.js

```diff
@@ -19,7 +19,7 @@
         paths: {
             "jquery-connections": baseURL + "jquery.connections.min",
             "jquery-timing": baseURL + "jquery-timing.min",
-            "d3": "/static/scripts/d3.min"
+            "d3": "https://d3js.org/d3.v6.min"
```

Comment:
Could this us the module from jsroot?
`"/static/modules/d3.mjs "`

Reply:
I do not think that ES6 module can be used here. 
One have to use alternative location